### PR TITLE
AbsComponent: self.reliability added as mandatory attribute

### PR DESCRIPTION
**Pull request by ntejos**
As stated. Test updated accordingly. I believe having this mandatory attribute is useful, particularly to integrate it with IGMGuesses and Joebvp.

### CODE REVIEW

File: linetools/isgm/abscomponent.py [hunk 1]
```diff
@@ -54,11 +54,17 @@ class AbsComponent(object):
...
+            'a' - reliable
+            'b' - possible
+            'c' - uncertain
+            'none' - not defined (default)
+    comment : str, optional
         A comment, default is ``
     """
     @classmethod
-    def from_abslines(cls, abslines, stars=None, **kwargs):
+    def from_abslines(cls, abslines, stars=None, reliability='none', **kwargs):
```
Comment (profxj) [inline linetools/isgm/abscomponent.py]: You want a string here or None ?

File: linetools/isgm/abscomponent.py [hunk 2]
```diff
@@ -127,10 +133,20 @@ def from_dict(cls, idict, coord=None, **kwargs):
             radec = SkyCoord(ra=idict['RA']*u.deg, dec=idict['DEC']*u.deg)
         # Init
         #slf = cls(radec, tuple(idict['Zion']), idict['zcomp'], Quantity(idict['vlim'], unit='km/s'),
+
+        # backwards compatibility
+        reliability = 'none'
```
Comment (profxj) [inline linetools/isgm/abscomponent.py]: Could put this in the if statement, if you wanted.

File: linetools/isgm/abscomponent.py [hunk 3]
```diff
@@ -183,7 +205,7 @@ def __init__(self, radec, Zion, zcomp, vlim, Ej=0./u.cm, A=None,
         # Optional
         self.A = A
         self.Ej = Ej
-        self.comment = comment
+        self.comment = str(comment)
```
Comment (profxj) [inline linetools/isgm/abscomponent.py]: This will not be Python2/3 compatible

File: linetools/isgm/abscomponent.py (hunk 1 above)
Comment (ntejos) [inline linetools/isgm/abscomponent.py]: String, the default will be 'none'.

**Comment by profxj** [comment]
Merging

**Pull request merged**